src/manifest.c

```diff
@@ -633,7 +633,6 @@ struct manifest *load_mom(int version)
 		printf("Failed to load %d MoM manifest\n", version);
 		goto out;
 	}
-	retried = false;
```

Comment:
So, it looks like my analysis may be off... Can you clarify when retries need to happen and when they do not?